ldap/servers/plugins/memberof/memberof.c

```diff
@@ -922,6 +927,7 @@ perform_needed_fixup()
         }
         be = slapi_get_next_backend(cookie);
     }
+    slapi_ch_free_string(&cookie);
```

Comment:
nice catch. you may cast with (void **)

Reply:
I think we generally free cookies like this, not very consistently...

We either do `slapi_ch_free_string(&cookie);` or `slapi_ch_free((void **)&cookie);` - which is the same:

https://github.com/389ds/389-ds-base/blob/d862f3a811ef8be275e5bb0cdc9d359c83a7376c/ldap/servers/slapd/ch_malloc.c#L305-L312

Reply:
You are right. I missed the '_string' . So just forget my comment